akka-docs/src/main/paradox/stream/operators/Sink/foreachAsync.md

```diff
@@ -0,0 +1,33 @@
+# foreachAsync
```

Comment:
I'm skeptical to if this adds much value over using `mapAsyncUnorderd` + `Sink.ignore`.
On the other hand we already have `foreachParallel`, which would have been better as `f: T => Future[Unit]` and we don't use the naming `Parallel` in other places.

Therefore it might be good to add this and deprecate `foreachParallel`.

Reply:
I think foreachParallel is for different purposes, although we can achieve the same `foreachParallel` behaviour as `f: T => Future[Unit]`  just by using the default dispatcher.

Reply:
if one need the same thing as `foreachParallel` one would use `foreachAsync(Future(f))`

Reply:
Have we reached a decision here?
I'm okey with keeping both in this PR. 
Potentially one could deprecate the Parallel version.

Reply:
I see it's deprecated now, all good then

Reply:
yes, I think we should be careful to not just add more and more to the API. Here it makes sense to trade one for the other.

`foreachAsync` is more consistent with other naming, and it's more powerful. `foreachParallel` can easily be expressed as `foreachAsync` by letting the user decide to spawn a Future or call out to something else.